scipy/stats/mstats_extras.py

```diff
@@ -392,7 +392,50 @@ def compare_medians_ms(group_1, group_2, axis=None):
         ndarray of floats with a length equal to the length of `group_1`
         along `axis`.
 
+    Examples
+    --------
+    >>> from scipy import stats
+    >>> a = [1, 2, 3, 4, 5, 6, 7]
+    >>> b = [8, 9, 10, 11, 12, 13, 14]
+
+    Arrays must be at least of size 7.
+
+    >>> stats.mstats.compare_medians_ms(a, b, axis=None)
+    1.0693225866553746e-05
+
+    Since no axis is provided, the result is flattened.
+
+    The McKean–Schrader estimate of the standard error is used 
+    in conjunction with a Welch-type test in hypothesis testing. 
+    It is used for the special case of testing the hypothesis of equal medians, 
+    when Yuen-Welch and Box methods are not recommended.
+    [1]_ Wilcox.
+
+    >>> c = [-5, 82, 11, 11, 22, 55, 90]
+    >>> stats.mstats.compare_medians_ms(b, c)
+    0.275814234966238
+
+    The function is vectorized to compute along a given axis.
+    Numpy Broadcasting compares the shapes of two arrays element-wise. 
+    Two dimensions are compatible when they are equal or one of them is 1.
+    [2]_NumPy Broadcasting.
+
+    >>> import numpy as np
+    >>> rng = np.random.default_rng()
+    >>> x = rng.random(size=(3, 7))
+    >>> y = rng.random(size=(3, 8))
+    >>> stats.mstats.compare_medians_ms(x, y, axis=1)
+    array([0.1718035 , 0.17301209, 0.40799719])
+
+    default_rng() is the recommended constructor for generating 
+    random numbers from a variety of probability distributions.
+
```

Comment:
```suggestion
```
Likewise, `default_rng()` is used in examples throughout the documentation. We currently assume some familiarity with NumPy.